src/risedevtool/src/task/postgres_service.rs

```diff
@@ -30,9 +30,16 @@ impl DockerServiceConfig for PostgresConfig {
 
     fn args(&self) -> Vec<String> {
         // Enable CDC.
-        ["-c", "wal_level=logical", "-c", "max_replication_slots=30"]
-            .map(String::from)
-            .to_vec()
+        [
+            "-c",
+            "wal_level=logical",
+            "-c",
+            "max_replication_slots=30",
+            "-c",
+            "log_statement=all",
```

Comment:
I want to log the PG statements. I think this is acceptable? cc @BugenZhao wdyt. Should we make this configurable instead, or it's fine to hardcode.

Reply:
LGTM